app/controllers/properties_controller.rb

```diff
@@ -0,0 +1,61 @@
+class PropertiesController < ApplicationController
+  def new
+    @property = current_user.properties.build
+  end
+
+  def create
+    @property = current_user.properties.build(property_params)
+     if @property.save 
+      redirect_to listing_property_url(@property)
+    else
+      render 'new'        
+    end
+    puts @property.errors.full_messages
+  end
+
+  def edit
+    @property = Property.find(params[:id])
+  end
+
+  def update
+    @property = Property.find(params[:id])
+    if @property.save
+      @property.update(property_params)
+      redirect_to request.referrer
+    end
+  end
+
+  def show
+  end
+
+  def listing
+    @property = Property.find(params[:id])
```

Comment:
Create a filter method for setting property: `@property = Property.find(params[:id])`